### ISSUE TITLE
git-flow

### ISSUE THREAD

**Issue by imaia** [olmokramer/atom-tree-view-git-branch#8]
git-flow support would be nice

**Comment by olmokramer** [comment]
Could you please be more specific? I've never worked with git-flow myself, and as far as I can tell it's more of a workflow kind of thing, so I don't really understand what it is that I should support.

**Comment by imaia** [comment]
That's right, it is a workflow kind of thing. Do you know jetbrains ides? They have a nice plugin where you can run git-flow commands easyly, from a combobox. Example: git-flow init or create a new feature. That's the kind of support that I believe would be nice.
